.github/stale.yml

```diff
@@ -0,0 +1,16 @@
+# Number of days of inactivity before an issue becomes stale
```

Comment:
I'll merge master into the kube-prometheus PR so this file doesn't appear the diff. Already committed in: https://github.com/scottrigby/prometheus-helm-charts/pull/27

Reply:
Done